docs/setup/native.md

```diff
@@ -74,16 +103,18 @@ All of the OSX instructions assume `homebrew` is your [package manager](https://
    - `brew install imagemagick`
 1. Install Poppler
    - `brew install poppler`
-1. Install ClamAV
+1. Install ClamAV or *use the fake scanner for development*
```

Comment:
I think we should default to using the fake clamav scanner. 99.9% of devs would prefer this. That means:

- use the fake one in `settings.yml`
- remove this "Install ClamAV" section altogether
  - maybe add some kind "optional steps" link to a separate markdown file on how to install Clam AV natively
- update the dev, stag, prod settings.yml files to override the fake clamav

Reply:
settings.yml isn't the best place -- development.yml is for exactly this kind of thing (with the option of settings.local).

Since we *also* hardcode the setting in the `docker-compose` file, i think untangling _there_ makes more sense and already on my agenda :)